app/scripts/modules/core/authentication/authentication.initializer.service.ts

```diff
@@ -5,11 +5,8 @@ import {IModalService, IModalStackService} from 'angular-ui-bootstrap';
 
 import {IDeckRootScope} from '../domain/deckRootScope';
 import {REDIRECT_SERVICE, RedirectService} from './redirect.service';
-import {AUTHENTICATION_SERVICE, AuthenticationService} from './authentication.service';
+import {AUTHENTICATION_SERVICE, AuthenticationService, IAuthResponse} from './authentication.service';
 
-interface IAuthResponse {
```

Comment:
It feels odd to pull this interface out of this service, since this service interacts with the API directly (whereas the Authentication Service does not). Is there a good reason for moving this interface that I'm not seeing?